templates/project/.cargo/config.toml

```diff
@@ -30,8 +30,3 @@ rustflags = ["-C", "default-linker-libraries"]
 
 build-std = ["std", "panic_abort"]
 #build-std-features = ["panic_immediate_abort"] # Required for older ESP-IDF versions without a realpath implementation
-
-[env]
-ESP_IDF_TOOLS_INSTALL_DIR = {value = "fromenv", force = true}
-ESP_IDF_VERSION= { value = "v4.4.1", force = true}
-MCU = { value= "esp32", force = true}
```

Comment:
I think this was actually wrong: above we set the triple based on the MCU that was selected, but then we overrode it here. As in the regular micro-rdk, we should let the triple selection drive the MCU.